js-exercises/alternating-characters/alternatingCharacters.js

```diff
@@ -0,0 +1,21 @@
+function alternatingCharacters(arrOfStrings) {
+  var alternatingCharactersCount = []
+	arrOfStrings.forEach(function(singleString) {
+		let counter = 0;
+     if (singleString.length >= 1 && singleString.length <= 100000) {
+     	let stringArray = singleString.split('');
+     	let past = stringArray[0];
+     	stringArray = stringArray.filter((item, key) => {
```

Comment:
As mentioned in clean code 2 class, foreach and filter can be decoupled to separate functions and name the functions appropriately for better readability.